fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -113,6 +113,25 @@ def self.is_test?
       self.test?
     end
 
+    def self.compare_paths(path1, path2)
+      compare_path1 = File.expand_path(path1)
+      compare_path2 = File.expand_path(path2)
+      if self.fs_is_insensitive?
+        # FS is insensitive, there fore the file `fastlane`  is also accessable via `FASTLANE``
```

Comment:
I don't understand that comment, also double space after fastlane, also `therefore` I think?